unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -294,6 +295,13 @@ abstract public class BaseFPSAgentController : MonoBehaviour {
                 Debug.LogError("ActionFinished called with agentState not in processing ");
             }
 
+            if (
```

Comment:
As well, would you want to check whether `lastActionInitialPhysicsSimulateCount` does not equal `PhysicsSceneManager.PhysicsSimulateCallCount` since `PhysicsSimulateCallCount` get incremented independent of `lastActionInitialPhysicsSimulateCount`?

Reply:
Hi @ekolve, we chatted a bit about this yesterday, was anything still unclear?